sdf/deployment.mdx

```diff
@@ -1,75 +1,68 @@
 ---
 title: Deployment
-description: Deployment/Worker Management
+description: Deployment of dataflow thru Worker
 sidebar_position: 60
 ---
 
-# Deployment/Worker Management
+# Introduction
 
-There are two modes to run dataflow in SDF: ephemeral and worker.  When [CLI] is run in ephemeral mode, it will run the dataflow in the same process as the CLI.  The dataflow will be terminated upon exiting CLI.  This is useful for development dataflow as well as testing the package.  
+When you use `run` command to execute a dataflow, it runs within the same process as the CLI. This is useful for development and testing because it's easy to start without needing to manage additional resources. It allows for quick testing and validation of the dataflow, and you can easily load and integrate development packages.
```

Comment:
```suggestion
When you use the `run` command to execute a dataflow, it runs within the same process as the CLI. This is useful for development and testing because it's easy to start without needing to manage additional resources. It allows for quick testing and validation of the dataflow, and you can easily load and integrate development packages.
```